src/pages/CommitDetailPage/Header/hooks/useCommitHeaderData.tsx

```diff
@@ -0,0 +1,139 @@
+import { useQuery } from '@tanstack/react-query'
+import { z } from 'zod'
+
+import {
+  RepoNotFoundErrorSchema,
+  RepoOwnerNotActivatedErrorSchema,
+} from 'services/repo'
+import Api from 'shared/api'
+import A from 'ui/A/A'
```

Comment:
`import A from 'ui/A/A'` should be `import A from 'ui/A'`

Reply:
Fixed